<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="5">
       <widget class="QComboBox" name="cbb_dw"/>
      </item>
      <item row="2" column="5">
       <widget class="QSpinBox" name="sb_sl"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="le_cpgg"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>录入日期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>数    量</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>产品规格</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>备    注</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="le_bzjg"/>
      </item>
      <item row="0" column="7">
       <widget class="QLineEdit" name="le_gys"/>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>单    位</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>标准价格</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>供 应 商</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>联系方式</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="le_lxfs"/>
      </item>
      <item row="2" column="7">
       <widget class="QLineEdit" name="le_bz"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>产品名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_cpmc"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="le_date"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="le_content"/>
      </item>
      <item>
       <widget class="QPushButton" name="pb_query">
        <property name="text">
         <string>产品查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_add">
        <property name="text">
         <string>产品添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_loadcsv">
        <property name="text">
         <string>导入CSV</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_cpmc</tabstop>
  <tabstop>le_cpgg</tabstop>
  <tabstop>le_bzjg</tabstop>
  <tabstop>le_gys</tabstop>
  <tabstop>le_lxfs</tabstop>
  <tabstop>le_bz</tabstop>
  <tabstop>le_content</tabstop>
  <tabstop>pb_query</tabstop>
  <tabstop>pb_add</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>cbb_dw</tabstop>
  <tabstop>le_date</tabstop>
  <tabstop>sb_sl</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
